### ISSUE TITLE
Debian package generation not working

### ISSUE THREAD

**Issue by gabrik** [eclipse-zenoh/zenoh#71]
Hi, 

Debian package generation is not working.

Running `cargo deb -p zenoh` gives:

```
$ cargo deb -p zenoh
warning: extended-description field missing. Using ../README.md, but markdown may not render well.
cargo-deb: unable to read extended description from file: ../README.md
  because: No such file or directory (os error 2)
 ```
 
 Instead is should build and generate the debian package.

**Comment by JEnoch** [comment]
The fix in #72 makes the publication of zenoh crate to fail:
```
 Uploading zenoh v0.5.0-beta.8 (/Users/genie.zenoh/workspace/zenoh-releases_master/zenoh)
error: failed to read `readme` file for package `zenoh v0.5.0-beta.8 (/Users/genie.zenoh/workspace/zenoh-releases_master/zenoh)`

Caused by:
  failed to read `/Users/genie.zenoh/workspace/zenoh-releases_master/zenoh/README.md`

Caused by:
  No such file or directory (os error 2)
```

**Comment by JEnoch** [comment]
Fixed via a symlink in commit 6e9ce13
